core/markets/storage/client/client_deal.hpp

```diff
@@ -6,9 +6,12 @@
 #pragma once
 
 #include "vm/actor/builtin/types/market/deal.hpp"
+#include "markets/storage/mk_protocol.hpp"
 
 namespace fc::markets::storage::client {
   using vm::actor::builtin::types::market::ClientDealProposal;
+  using libp2p::peer::PeerInfo;
+  using primitives::DealId;
```

Comment:
What is a reason to add this "include" and these "usings" to the file?

Reply:
Because these types were used in this files, but there were not include/using.  
This file compiled only because it was included once from other file which included these includes **before** this file.

Reply:
Ok. Reviewed on phone, and didn't see whole file